spec/cucumber/formatter/junit_spec.rb

```diff
@@ -2,42 +2,41 @@
 
 require 'spec_helper'
```

Comment:
This is a crazily complex file, so I've not reviewed it.

Reply:
This is actually the same kind of changes as with http_io_spec and configuration_spec. Nothing else